ui/lib/apps/ContinuousProfiling/pages/Detail.tsx

```diff
@@ -26,26 +37,63 @@ export default function Page() {
 
   const profileDuration = groupProfileDetail?.profile_duration_secs || 0
 
+  const [tableData, groupData] = useMemo(() => {
+    const newRows: ConprofProfileDetail[] = []
+    const newGroups: IGroup[] = []
+
+    let startIndex = 0
+    const profiles = groupProfileDetail?.target_profiles || []
+    // rename profile to cpu for profile_type for easier sort
+    profiles.forEach((p) => {
+      if (p.profile_type === 'profile') {
+        p.profile_type = 'cpu'
+      }
+    })
```

Comment:
Can we sort profile type by specific enum order.

Reply:
How can we do it, do you have an example?

Reply:
Do we need to specify the ordering here? Maybe I misunderstood it before.

Reply:
Yep, the design spec requires.

Reply:
Maybe keeping a specific order is fine. There is no need to make CPU the first one? Please confirm with PM.

Even if you want to sort CPU as the first, what you should do is defining a sort key array and sort by the sort key.

Reply:
Confirmed with PM, it is required, let me refine the sort method.

Reply:
done!